exercises/phone-number/.meta/template.j2

```diff
@@ -0,0 +1,30 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% set class = exercise | camel_case -%}
+{{ macros.header([class]) }}
+
+class {{ class }}Test(unittest.TestCase):
+    {% for case in cases[0]["cases"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if "error" in case["expected"] -%}
+        with self.assertRaisesWithMessage(ValueError):
+            {{ class }}("{{ case["input"]["phrase"] }}")
+        {% else -%}
+        number = {{ class }}("{{ case["input"]["phrase"] }}").number
+        self.assertEqual(number, "{{ case["expected"] }}")
+        {% endif %}
+    {% endfor %}
+
+    # Track specific tests
+    def test_area_code(self):
```

Comment:
These seem pretty simple to generate from an `additional_tests.json` file.

Reply:
I add the `additional_tests.json` but I can't generate the test file from it with the same command. Do I have to change something with the `--spec-file`?

Reply:
Never mind. I fixed that.